app/shared/components/paper-component/PaperComponent.styles.ts

```diff
@@ -0,0 +1,42 @@
+import { mainHexPallete } from '../design-system/all-components/theme/colors';
+
+export const styles = {
+  container: {
+    backgroundColor: mainHexPallete.white,
+    padding: {
+      xs: '40px 24px',
+      sm: '64px 32px',
+      md: '76px 53px',
+      lg: '90px 80px'
+    },
+    width: '100%',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center'
```

Comment:
```suggestion
    alignItems: 'center',
     boxShadow: 'none'
```
there is no box shadow in our design, so we can remove it

Reply:
> there is no box shadow in our design, so we can remove it

Ok, I think it might be better to use the elevation prop of the Paper component to remove the shadow.